README.md

```diff
@@ -13,6 +13,12 @@ For getting mp4 playback to work, I've setup the conda environment with
 conda install -c loopbio -c conda-forge -c pkgw-forge ffmpeg gtk2 opencv
 ```
 
+On *Windows* you can ommit the `gtk2` package when using [Anaconda 5.3 Windows Installer](https://www.anaconda.com/download/)
+```
+conda install -c loopbio -c conda-forge -c pkgw-forge ffmpeg gtk2 opencv
```

Comment:
Why didn't you omit it then? :slightly_smiling_face: 
I think we can "omit" line 18, and just leave the text remark :wink: